index.html

```diff
@@ -175,5 +176,82 @@ <h3 class="tileone">Skills</h3>
         <img class="clipart" src="bulb.png" alt="bulb" />
       </div>
     </section>
+
+    <section class="Forms">
+      <form action="index.html" class="forms">
```

Comment:
Great job with the styling. 
- [ ] Please the Form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html)